askbot/deps/django_authopenid/views.py

```diff
@@ -178,7 +181,7 @@ def logout_page(request):
         'page_class': 'meta',
         'have_federated_login_methods': util.have_enabled_federated_login_methods()
     }
-    return render(request, 'authopenid/logout.html', Context(data))
```

Comment:
You're welcome, I removed Context while following [this](http://stackoverflow.com/questions/26072460/valueerror-after-upgrading-django-from-1-6-to-1-7) to remove the ValueError that I was getting